### ISSUE TITLE
Publishung to ArtifactHub.io

### ISSUE THREAD

**Issue by venkatamutyala** [metacontroller/metacontroller#748]
Hi, any reason this hasn't been formerly published to artifacthub.io?  There are a couple of third party versions but i think it would be nice to have one straight from you folks. I imagine this would help increase adoption of this tool as well

Third party: https://artifacthub.io/packages/search?ts_query_web=metacontroller&sort=relevance&page=1

**Comment by grzesuav** [comment]
hi @venkatamutyala , it shit about docker images or helm charts (or both)?

**Comment by fernandoataoldotcom** [comment]
hey @grzesuav - sorry for our delay!
We've built some tools using Metacontroller and plan for more - thanks a lot for the project.

This is for the helm charts.  We're looking at using metacontroller for our platform and see a few packages that have been published: (OpenStack)[https://artifacthub.io/packages/helm/openstack-helm/metacontroller], (CowboySysOp)[https://artifacthub.io/packages/helm/cowboysysop/metacontroller?modal=values], but neither of them look up to date or officially provided by Metacontroller.

Would you like our help setting up a pipeline to publish an official chart under a Metacontroller account in ArtifactHub?

**Comment by grzesuav** [comment]
it seems to be kind of proxy service, registered myself and added a repo, probably it will be processed in some time

**Comment by venkatamutyala** [comment]
Yes, but you do need to publish the helm chart someplace. You should be able to do this on github pages pretty easily. Let us know if you need any help setting this up. Here is an example: https://www.harness.io/blog/helm-chart-repo

I believe the repository you publish to cannot be an OCI repo. I think artifacthub requires the index.yaml but I could be mistaken.

**Comment by grzesuav** [comment]
https://artifacthub.io/packages/search?repo=metacontroller&sort=relevance&page=1

**Comment by grzesuav** [comment]
I will try to adjust it a bit to be more informative (add readme, mark as official etc) hopefully you can use it as they for time being ? @venkatamutyala @fernandoataoldotcom

**Comment by venkatamutyala** [comment]
@grzesuav the OCI doesn't work very well with tools like ArgoCD. It might be best for this issue to become creating a non-OCI chart. We are happy to help set this up as well if you folks would like.

**Comment by venkatamutyala** [comment]
Seems like there could be a path within github:
https://stackoverflow.com/questions/15563685/can-i-create-more-than-one-repository-for-github-pages

**Comment by grzesuav** [comment]
The one within github requires multiple repositories, I might extract the helm chart to separate repo however was hoping for easier solution. I was also hoping to see more hosting sites for raw helm charts - there is a bunch for OCI registries.

**Comment by grzesuav** [comment]
@mikesmithgh I think having helm charts in separate repo is more viable option - I can trigger helm build once in reglar repository new version will be created. Also we can do fixes independently in helm repo - for chelm charts

**Comment by venkatamutyala** [comment]
@grzesuav @mikesmithgh is this something I'd be able to help push forward?

**Comment by grzesuav** [comment]
hi @venkatamutyala , I haven't been able to figure out where to place helm charts. One thing which I am considering is to use gitlab chelm repository - https://docs.gitlab.com/ee/user/packages/helm_repository/

**Comment by venkatamutyala** [comment]
@grzesuav do you folks get any free sponsorship via AWS/GCP/etc? If so i'd be happy to help you folks get pipelines setup in a cloud. otherwise, it appears that gitlab option looks OK but since you already have a github org, there is the option to use github pages: https://medium.com/@gerkElznik/provision-a-free-personal-helm-chart-repo-using-github-583b668d9ba4

Additonal ref: https://www.truefoundry.com/blog/hosting-helm-charts-on-github-pages
